### PR TITLE
Document newsletter processes

### PR DESCRIPTION
**Pull request by vdauwera**

### CODE REVIEW

File: editorial-process.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+_This doc aims to document the process by which the newsletter is created and distributed, mainly for the benefit of new volunteer editors._
+
+## Process for publishing a new issue of the newsletter
+
+1. When you start a new issue of the newsletter, there should already be an open github issue open calling for contributions from the community (created before the previous newsletter was sent out). Make sure to disseminate the link widely on social media.
```
Comment (nlharris) [inline editorial-process.md:5]: an open github issue open -> a GitHub issue open

File: editorial-process.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+2. Create a NEW github issue (using the default issue template) with the appropiate number for the NEXT issue of the newsletter. (Try not to confuse github issue numbers and newsletter issue numbers.) This will be necessary to update relevant links in the newsletter itself. 
+3. Create a new branch named `newsletter-YYYY-MM` and make a copy of the `newsletter-template.md` file, renaming it to `YYYY-MM.md` and moving it to the `newsletters/` directory.
+4. Update the Table of Contents (`table-of-contents.md`) to link to the new newsletter file, and add a line linking to the new github issue you created earlier for the next newsletter. Add a new year subheading if needed.
+5. In the `YYYY-MM.md` file, edit the `FIX THIS LINK` instances in the header notes (link to newsletter file in github) and in the footer notes (link to the call for contributions for the next issue; again this is the new github issue you created earlier). 
+6. Start filling in the relevant content by section. A few notes:
+  - Ask for any OBF and GSoC updates in the OBF slack workspace (open-bio.slack.com, distinct from the BOSC slack workspace);
```
Comment (nlharris) [inline editorial-process.md:11]: Also tweet as @OBF_NEWS

**Review by nlharris — APPROVED** [review]
This is so helpful; thanks for putting this together!

**Comment by vdauwera** [comment]
Thanks @nlharris ! I'll leave this open until I complete this newsletter cycle, as I'm sure I'll need to add items/amendments as it all plays out.

File: editorial-process.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+5. In the `YYYY-MM.md` file, edit the `FIX THIS LINK` instances in the header notes (link to newsletter file in github) and in the footer notes (link to the call for contributions for the next issue; again this is the new github issue you created earlier). 
+6. Start filling in the relevant content by section. A few notes:
+  - If lacking community updates, get @OBF_NEWS to tweet out the call again;
+  - Ask for any OBF and GSoC updates in the OBF slack workspace (open-bio.slack.com, distinct from the BOSC slack workspace);
+  - The Event Fellowships section (formerly Travel Fellowships) is a standard blurb, you just need to update the event and deadline dates.
+  - [...? Anything else to note?]
+7. When the draft is complete, open a PR and ask OBF board members (via OBF slack) to review/comment. Iterate until approval. 
+8. Once the draft has been approved, close the issue calling for contributions and merge the PR.
+9. Send the newsletter out via the mailing lists; (THIS BIT TBD SINCE I HAVEN'T DONE IT YET).
```
Comment (peterjc) [inline editorial-process.md:17]: Main README for the repository says:

> We are sending it through the [bosc-announce](https://groups.google.com/g/bosc-announce/) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists.

BOSC announce is now via Google Groups, that likely needs approval (of each message, or of the sender as a one-off).

open-bio-l is still on mailman, and would require the sender be subscribed first. The message might still be held for moderation.

Either way, dropping the OBF board an email to confirm both messages are en route and potentially need moderating should suffice.

Comment (peterjc) [reply editorial-process.md:17]:
```suggestion
9. Send the newsletter out via the mailing lists, bosc-announce (Google Groups will require moderation for first time) and open-bio-l (mailman requires subscribing first). Forward to the OBF Board noting mailing list moderation may be required.
```

Comment (nlharris) [reply editorial-process.md:17]: (not sure how to suggest a change to your suggested change!)

i think this is fine, but the last line is a bit hard to parse. i'd add a few words:
Forward the newsletter to the OBF Board, alerting them that mailing list moderator approval may be required.

**Review by peterjc — APPROVED** [review]
